### DISCUSSION TITLE
ADA vs Ada - uppercase vs lowercase

### DISCUSSION THREAD

**Discussion by RayTracedObserver** [cardano-foundation/cardano-org#90] [General]
Is there any particular reason why most mentions of Ada on the website are written in lowercase rather than using the uppercase ADA that is seen on most of the Internet ? Coin ticker symbols, just like stock ticker symbols, seem to be always written uppercase.

While Ada is named after Ada Lovelace, it seems like switching all instances of Ada into uppercase ADA would perhaps be desirable to make a distinction between the person and the token. It would be a way to match the common standard of ticker names being uppercase.

The only places where Ada would remain lowercase on the website are the few mentions of the person.

**Comment by katomm** [comment]
Yes. When talking about the cryptocurrency, do not capitalize, unless at the beginning of a sentence. The idea behind this is to treat it like dollars or euros. If you are in doubt, in English, prefer ada over ADA. Capitalised ADA stands for the ticker symbol only. (Like EUR/USD)

**Reply by RayTracedObserver** [reply]
Ok thank you for the clarification.
